src/com/magento/idea/magento2plugin/util/magento/ComponentType.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+package com.magento.idea.magento2plugin.util.magento;
+
+public class ComponentType {
+    public static String TYPE_MODULE = "module";
+    public static String TYPE_THEME = "theme";
```

Comment:
Hi @pmclain 

Let's convert constants to an enum (https://www.w3schools.com/java/java_enums.asp). Enums are used for polymorphism and make the code safer. 
Thank you!

Reply:
I'd move this to `com/magento/idea/magento2plugin/magento/packages`.